docs/conf.py

```diff
@@ -0,0 +1,79 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+# Tools for importing snscrape at build time
+# Avoid name conflict with sphinx configuration variable "version"
+from importlib import import_module
+from importlib.metadata import version as _version
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'snscrape'
+copyright = '2021, JustAnotherArchivist'
```

Comment:
Should this be updated to 2022?

Reply:
This should be derived from the version number retrieved via `importlib.metadata` instead of being hardcoded. Cf. my first comment on this PR.